index.js

```diff
@@ -222,6 +223,19 @@ class DeltaChat extends EventEmitter {
     this.emit('ALL', event, data1, data2)
 
     const eventStr = events[event]
+    const self = this
+
+    async function handleHttpGetEvent (url) {
+      try {
+        debug('handleHttpGetEvent url', url)
+        const response = await got(url, {})
+        debug('handleHttpGetEvent response.body', response.body)
+        binding.dcn_set_http_get_response(self.dcn_context, response.body)
+      } catch (error) {
+        debug('handleHttpGetEvent error.response.body', error.response.body)
+        binding.dcn_set_http_get_response(self.dcn_context, '')
```

Comment:
**Note** I'm currently passing in an empty string. We could use this in the c-handler to return `0` if the string is empty.